### PR TITLE
[KYUUBI #3437] Refactory class location of the hive connector

### PR DESCRIPTION
**Pull request by Yikf**
### _Why are the changes needed?_

Fix https://github.com/apache/incubator-kyuubi/issues/3437

This pr aims to refactory class location of the hive connector

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/spark/sql/hive/kyuubi/connector/HiveConnectorBridgeHelper.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ import org.apache.spark.sql.hive.{HadoopTableReader, HiveShim, HiveTableUtil}
 import org.apache.spark.sql.hive.client.HiveClientImpl
 import org.apache.spark.sql.types.{DataType, DoubleType, FloatType, StructType}
 
-object HiveConnectorHelper {
+object HiveConnectorBridgeHelper {
```
Comment (pan3793) [inline extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/spark/sql/hive/kyuubi/connector/HiveConnectorBridgeHelper.scala:33]: how about `HiveBridgeHelper`?

**Comment by pan3793** [comment]
As now the connector has read/write capabilities, let's port Hive V1 test suites to make sure that V2 implementation has the same behavior or reasonably changes.

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

File: extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/HiveTable.scala [hunk 2]
```diff
@@ -31,10 +31,11 @@ import org.apache.spark.sql.connector.catalog.TableCapability.{BATCH_READ, BATCH
 import org.apache.spark.sql.connector.expressions.Transform
 import org.apache.spark.sql.connector.read.ScanBuilder
 import org.apache.spark.sql.connector.write.{LogicalWriteInfo, WriteBuilder}
-import org.apache.spark.sql.hive.kyuubi.connector.HiveConnectorHelper.{logicalExpressions, BucketSpecHelper}
+import org.apache.spark.sql.hive.kyuubi.connector.HiveBridgeHelper.{logicalExpressions, BucketSpecHelper}
```
Comment (yaooqinn) [inline extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/HiveTable.scala:34]: what not kyuubi.hive.connnector?

**Comment by pan3793** [comment]
> ... or all the scenarios that need validation ...

We should cover cases as much as possible

File: extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/HiveTable.scala (hunk 2 above)
Comment (pan3793) [reply extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/HiveTable.scala:34]: Sorry, I missed this comment, given the CI is queued, I verified the PR locally and merged it to unblock the following changes.

**Pull request closed without merging**